OceanProject.uproject

```diff
@@ -9,5 +9,11 @@
 			"Type": "Runtime",
 			"LoadingPhase": "Default"
 		}
+	],
```

Comment:
Why are we referencing our self in this plugin? Is this for the engine module compatibility? Does this still work as a standalone plugin?

Reply:
Yes of course, I'm not sure how the project got away with this before xD Thanks for catching that